### ISSUE TITLE
selfcompiled.ttf

### ISSUE THREAD

**Issue by idontveapc** [popemkt/segoe-ui-emoji#2]
they're blank...
![Screenshot_20231219-114739](https://github.com/popemkt/segoe-ui-emoji/assets/122597295/be2ce191-63b2-4a77-a935-e83910860007)
![Screenshot_20231219-114746](https://github.com/popemkt/segoe-ui-emoji/assets/122597295/36458619-4418-48d2-919d-3a18f1dafb3e)

**Comment by popemkt** [comment]
Yes for some reasons they're blank when previewing in certain softwares, but when testing on https://wakamaifondue.com/ there is all the char set.
And I'm also using it on my OS, so should be good.

**Comment by idontveapc** [comment]
I'm on a11 and this supports colrv0 but when using 3d; the bnw glyphs are always used:

**Comment by idontveapc** [comment]
![Screenshot_20231219-120537](https://github.com/popemkt/segoe-ui-emoji/assets/122597295/5d828a8f-a313-4a14-b5c2-312c1f97eb69)

**Comment by idontveapc** [comment]
flat ones are colrv0 but the selfcompiled.ttf might uses different svgs!? they're blank on my apps

**Comment by idontveapc** [comment]
selfcompiled on browser viewing emojipedia.org:
![ResizedImage_2023-12-19_12-26-52_7110](https://github.com/popemkt/segoe-ui-emoji/assets/122597295/be2aa35b-cce5-47f9-9dbb-0f6606207f21)

**Comment by popemkt** [comment]
Yeah my mistake on the naming, it should be self-compiled-flat

**Comment by idontveapc** [comment]
yup, but sad new changed designs (like the eye emoji) are still like from the old font, but you have new u15 emojis though.

**Comment by popemkt** [comment]
I noticed that too (from articles on emojipedia), but that's good enough to me for now, once the u15 branch is officially merged I'll do the compile again (maybe with proper versioning and pipeline).
Anw thanks for testing it out.

**Comment by idontveapc** [comment]
i think im unlucky, can't have either 3d or flat anymore :(

**Comment by popemkt** [comment]
That's strange, I could have only flat if I override the current version (my current version is 1.45) with 1.35 on windows, the compiled one I have only tested on my fedora machine.

**Comment by idontveapc** [comment]
yup the old (1.35) flat emojis work but they u14:
![Screenshot_20231219-124438](https://github.com/popemkt/segoe-ui-emoji/assets/122597295/02047b82-285c-4e25-a21b-756ad2c01863)

**Comment by idontveapc** [comment]
on windows 11, i think program with colrv1 support would show 3d. it's weird why windows 11 itself not supporting colrv1? noto colrv1 must work on win11!?

**Comment by popemkt** [comment]
yeah kinda wacky in windows case. On the side note, I can see that v1.45 has a few times more glyphs than earlier versions, that would mean they have alternate glyphs for colrv1 (gradient) and non-colrv1 (flat) or am I mistaking anything?

**Comment by idontveapc** [comment]
yup like melting-face was flat before but now it's showing 3d too.

and some bnw hand gestures (with Fitzpatrick) emojis were missing, they added.

**Comment by idontveapc** [comment]
![ResizedImage_2023-12-19_12-53-02_5208](https://github.com/popemkt/segoe-ui-emoji/assets/122597295/0300390e-d070-4534-9172-4eb4d1e3f7f1)
hope it wouldn't roast win people.

**Comment by idontveapc** [comment]
a13 and a14 people with root can use and have them on any app.

**Comment by idontveapc** [comment]
[this script](https://github.com/Download/fluentui-emoji/tree/generate-font/scripts/generate-fonts) was a few days ago: https://github.com/Download/fluentui-emoji/tree/generate-font/scripts/generate-fonts

**Comment by popemkt** [comment]
That script yields pretty gnarly result tbh 
![image](https://github.com/popemkt/segoe-ui-emoji/assets/35656480/bd482b19-83e4-41e4-b79f-f6bd3511e91c)
But I don't think it's necessary, the [authentic PR](https://github.com/microsoft/fluentui-emoji/pull/113) already incorporates fixes for the assets metadata, which is the main issue, regardless of the method for compiling. Once merged should be good, although it should be noted that the repo lacks certain assets for say flags, many ZWJ combinations, etc. as compared to the font shipped with Windows.  Compilation won't fix that unfortunately.

**Comment by idontveapc** [comment]
why they distorted?
